doc/cookbook/source/examples/converter/multidimensional_scaling.rst

```diff
@@ -0,0 +1,54 @@
+========================
+Multidimensional Scaling
+========================
+
+Multidimensional Scaling (MDS) is a set of data analysis methods, which allow one to infer the dimensions of the perceptual space of subjects. The entering data are typically a measure of the global similarity or dissimilarity of objects, called proximities. The outcome is a spatial configuration in which the objects are represented as points arranged in such a way, that their distances correspond to their proximities.
```

Comment:
Please put every sentence in a new line. This is a new requirement we added that makes later diffs easier to read.